packages/gestalt/src/ExpandableModule.js

```diff
@@ -0,0 +1,45 @@
+// @flow strict
+import React, { useState, type Node } from 'react';
+import Box from './Box.js';
+import Divider from './Divider.js';
+import ExpandableModuleBase from './ExpandableModuleBase.js';
+import styles from './ExpandableModule.css';
+
+type Props = {|
+  items: Array<{|
+    title: string,
+    icon?: string,
+    iconAccessibilityLabel?: string,
+    summary?: Array<string>,
+    type?: string,
```

Comment:
type should have options? Im not sure what the different types are:

`type: "error" | "warning" | "info"`  maybe?

Reply:
Agreed with @simkessy - ideally we would be more strict

Reply:
Updated.
But we don't have a design for Warning state, so I put
type: "error" | "info" for now